container.go

```diff
@@ -99,6 +99,16 @@ func (c *ContainerConfig) valid() bool {
 	return true
 }
 
+// SystemMountsInfo describes additional information for system mounts that the agent
+// needs to handle
+type SystemMountsInfo struct {
+	// Indicates if /dev has been passed as a bind mount for the host /dev
+	BindMountDev bool
+
+	// Size of /dev/shm assigned by docker.
```

Comment:
No need to mention Docker from this comment. I would say `Size of /dev/shm assigned by the container specification.`

Reply:
fixed